test/system/200-pod.bats

```diff
@@ -199,6 +212,34 @@ function random_ip() {
     run_podman pod ps --no-trunc --filter "label=${labelname}=${labelvalue}" --format '{{.ID}}'
     is "$output" "$pod_id" "pod ps --filter label=..."
 
+    # Test local port forwarding, as well as 'ps' output showing ports
+    # Run 'nc' in a container, waiting for input on the published port.
+    c_name=$(random_string 15)
+    run_podman run -d --pod mypod --name $c_name $IMAGE nc -l -p $port_in
+    cid="$output"
+
+    # Try running another container also listening on the same port.
+    run_podman 1 run --pod mypod --name dsfsdfsdf $IMAGE nc -l -p $port_in
+    is "$output" "nc: bind: Address in use" \
```

Comment:
This might be fragile since it relies on the output of `nc`, a tool we don't own